operator/controllers/semconv/keptn.go

```diff
@@ -0,0 +1,9 @@
+package semconv
```

Comment:
It might be time to move such things to a /pkg folder of the lifecycle controller ... what do you think?

Reply:
The comment of @bacherfl has strong motivation for not going to a common pkg: https://github.com/keptn-sandbox/lifecycle-controller/pull/115#discussion_r987586292